etc/config/wasm.defaults.properties

```diff
@@ -0,0 +1,21 @@
+compilers=&wasmtime
```

Comment:
The `.defaults` ids and groups should be different from the `amazon` ones, it would conflict if not!